src/components/DisplayEmbed/DisplayExternal.jsx

```diff
@@ -132,6 +132,26 @@ export class DisplayExternal extends Component {
       domain,
     } = this.state;
 
+    const errorHolder = () => (
+      <>
+        <DeleteButton stripped onClick={onRemoveClick} />
+        <EditorErrorMessage
+          msg={
+            error
+              ? t('displayOembed.errorMessage')
+              : t('displayOembed.notSupported', {
+                  type,
+                  provider,
+                })
+          }
+        />
+      </>
+    );
+
+    if (error) {
+      return errorHolder();
+    }
+
     if (!type && !provider) {
       return null;
     }
```

Comment:
Eneste grunnen jeg ser til at denne må være der er for å tilfredsstille testen😬 for den gir `TypeError: context.formatMessage is not a function` hvis den prøver å displaye errormeldingen, selv om testrendereren sender inn en enkel funksjon for t (`t={() => 'Error message'}`) som prop.

Reply:
Det hadde nok vore bedre å fikse dette slik at feilmeldinga faktisk vises i testen. At testen skal forvente å få null gir liten verdi.

Reply:
Ja enig, men vil ikke det ligge på snapshot? For den tryner før den sjekker snapshottet fordi den ikke klarer å lage de feilmeldingene med bruk av t.

Reply:
Ja, så testoppsettet bør fikses slik at t fungerer.